lib/Bridge/Phpactor/MemberProvider/MixinMemberProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Phpactor\WorseReflection\Bridge\Phpactor\MemberProvider;
+
+use Phpactor\WorseReflection\Core\Reflection\Collection\ReflectionMemberCollection;
+use Phpactor\WorseReflection\Core\ServiceLocator;
+use Phpactor\WorseReflection\Core\Virtual\Collection\VirtualReflectionMemberCollection;
+use Phpactor\WorseReflection\Core\Virtual\ReflectionMemberProvider;
+use Phpactor\WorseReflection\Core\Reflection\ReflectionClassLike;
+use Phpactor\WorseReflection\Core\Reflection\ReflectionMember;
+
+class MixinMemberProvider implements ReflectionMemberProvider
+{
+    public function provideMembers(ServiceLocator $locator, ReflectionClassLike $class): ReflectionMemberCollection
+    {
+        $collection = VirtualReflectionMemberCollection::fromMembers([]);
+        foreach ($class->docblock()->mixins() as $name) {
+            $fqn = $class->scope()->resolveLocalName($name);
+            $mixinClass = $locator->reflector()->reflectClass($fqn);
+
+            $collection = $collection->merge($mixinClass->members());
```

Comment:
note this is wrong. this will add the actual members to the class and make no distinction between the "real" methods and the fake ones.

we will need to convert the actual member to a VirtualMember.

Reply:
also handle class-not-found